src/index.ts

```diff
@@ -484,7 +493,12 @@ export default class AbstractBtpPlugin extends EventEmitter2 {
       this._incomingWs.close()
       this._incomingWs = undefined
     }
-    if (this._wss) this._wss.close()
+    if (this._wss) {
+      const wss = this._wss
+        if (wss.options && wss.options.server)
+          wss.options.server.close()
```

Comment:
The http server should just be stored as a field of the plugin instead of reaching into the websocket to get it